src/pages/login/login-component.tsx

```diff
@@ -56,39 +59,62 @@ export class LoginComponent extends React.Component<LoginComponentProperties> {
     );
   }
 
-  renderFooter(stage) {
+  renderFooter(stage: LoginStage) {
     return (
       <div {...cn('other')}>
         <span>
-          Not on ZERO? <Link to='/get-access'>Create account</Link>
+          {(stage === LoginStage.EmailLogin || stage === LoginStage.Web3Login) && (
+            <>
+              Not on ZERO? <Link to='/get-access'>Create account</Link>
+            </>
+          )}
+          {stage === LoginStage.EmailLogin && (
+            <div {...cn('forgotten-password-container')}>
+              Forgot your password?{' '}
+              <button {...cn('button-link')} onClick={() => this.props.handleSelectionChange('reset')}>
+                Reset
+              </button>
+            </div>
+          )}
+          {stage === LoginStage.ResetPassword && (
+            <>
+              Back to{' '}
+              <button {...cn('button-link')} onClick={() => this.props.handleSelectionChange('backToLogin')}>
+                Log in
+              </button>
+            </>
+          )}
         </span>
-        {stage === LoginStage.EmailLogin && (
-          <span>
-            Forgot your password? <Link to='/get-access'>Reset</Link>
-          </span>
-        )}
       </div>
     );
   }
 
   render() {
     const { isLoggingIn, stage } = this.props;
     const isWeb3LoginStage = stage === LoginStage.Web3Login;
+    const isResetPasswordStage = stage === LoginStage.ResetPassword;
     const selectedOption = isWeb3LoginStage ? 'web3' : 'email';
 
     return (
       <>
         <ThemeEngine theme={Themes.Dark} />
         <div {...cn('')}>
-          <main {...cn('content')}>
+          <main {...cn('content', isResetPasswordStage && 'is-reset-stage')}>
```

Comment:
```suggestion
          <main {...cn('content', isResetPasswordStage && 'reset-stage')}>
```
I don't think the classname shoud also be `is-reset-stage`

Reply:
Is it possible to expand on why? Happy to change, but I'm not sure `reset-stage` reflects the condition in the styles as well as using `is`.  I think it mirrors the condition `isResetPasswordStage` clearly, but again, happy to change if that is misleading.